src/config.ts

```diff
@@ -224,34 +224,26 @@ export function getDefaultConfig(): IConfig {
     return Config.util.cloneDeep(defaultConfig);
 }
 
-/**
- * Grabs an explicit path provided for mjolnir's config from an arguments vector if provided, otherwise returns undefined.
- * @param argv An arguments vector sourced from `process.argv`.
- * @returns The path if one was provided or undefined.
- */
-function configPathFromArguments(argv: string[]): undefined | string {
-    const configOptionIndex = argv.findIndex(arg => arg === "--draupnir-config");
-    if (configOptionIndex > 0) {
-        const configOptionPath = argv.at(configOptionIndex + 1);
-        if (!configOptionPath) {
-            throw new Error("No path provided with option --draupnir-config");
-        }
-        return configOptionPath;
-    } else {
-        return;
-    }
-}
-
 export function read(): IConfig {
-    const explicitConfigPath = configPathFromArguments(process.argv);
```

Comment:
You shouldn't have erased explicitly passing the arguments vector to this function. It was done so that it would be possible to test the function.

Reply:
Can you really attach tests to non-exported functions though?

Reply:
Good observation, but it's still better to use explicit arguments over global variables where possible. It will make things easier for the future.